azure-pipelines.yml

```diff
@@ -64,14 +64,12 @@ jobs:
       testResultsFiles: '**\*.trx'
     condition: succeededOrFailed()
 
-# Linux build does not work when we mix runtimes and 
-# we don't use the results to do anything, skipping it for now  
-# - job: Linux
-#   pool:
-#     vmImage: 'ubuntu-16.04'
-#   variables:
-#       buildConfiguration: 'Release'
-#   steps:
-#   - script: ./build.sh -c $(buildConfiguration)
-#     displayName: './build.sh -c $(buildConfiguration)'
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
```

Comment:
Should this be changed to 'ubuntu-18.04' as well ? I see you already updated the TARGET_RUNTIME in the build script.

Also, taking into account Ubuntu's LTS schedule, version 16.04 should be running out of support in <1 year.

Reply:
I thought I changes it there 😁 Fixed